src/virtual_memory/anonymous.cpp

```diff
@@ -4,17 +4,18 @@
  * @copyright Eta Scale AB. Licensed under the Eta Scale Open Source License. See the LICENSE file for details.
  */
 
-#include <cerrno>
-#include <cstddef>
+// C headers
 #include <fcntl.h>
-#include <iostream>
-#include <string>
 #include <sys/mman.h>
 #include <sys/stat.h>
 #include <sys/statvfs.h>
 #include <sys/types.h>
-#include <system_error>
```

Comment:
header is still used in this file

Reply:
It's really not needed, but put back anyway.